networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -73,7 +95,7 @@ You can now export eBPF-enriched network flows to the IPFIX collector. For more
 
 [id="network-observability-packet-drop-1.4"]
 ==== Packet drops
-In the 1.4 release of the Network Observability Operator, eBPF tracepoint hooks are used to enable packet drop tracking. You can now detect and analyze the cause for packet drops and make decisions to optimize network performance. This feature is only supported in {product-title} versions 4.13+. For more information, see xref:../../networking/network_observability/observing-network-traffic#network-observability-pktdrop-overview_nw-observe-network-traffic[Configuring packet drop tracking] and xref:../../networking/network_observability/observing-network-traffic#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
+In the 1.4 release of the Network Observability Operator, eBPF tracepoint hooks are used to enable packet drop tracking. You can now detect and analyze the cause for packet drops and make decisions to optimize network performance. In 4.14+, both host drops and OVS drops are detected. In 4.13, only host drops are detected. For more information, see xref:../../networking/network_observability/observing-network-traffic#network-observability-pktdrop-overview_nw-observe-network-traffic[Configuring packet drop tracking] and xref:../../networking/network_observability/observing-network-traffic#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
```

Comment:
To be clear on what product we are talking about
```suggestion
In the 1.4 release of the Network Observability Operator, eBPF tracepoint hooks are used to enable packet drop tracking. You can now detect and analyze the cause for packet drops and make decisions to optimize network performance. In {product-title} 4.14 and later, both host drops and OVS drops are detected. In {product-title} 4.13, only host drops are detected. For more information, see xref:../../networking/network_observability/observing-network-traffic#network-observability-pktdrop-overview_nw-observe-network-traffic[Configuring packet drop tracking] and xref:../../networking/network_observability/observing-network-traffic#network-observability-packet-drops_nw-observe-network-traffic[Working with packet drops].
```

Reply:
These changes are actually part of https://github.com/openshift/openshift-docs/pull/66047...they won't be relevant until 4.14 so I'm going to redact them here. Not sure actually how they ended up here